src/globals.h

```diff
@@ -40,7 +40,27 @@
 #define BATTERY_MONITOR BAT_INTERNAL
 #endif
 
-#if defined(LED_INVERTED) && LED_INVERTED
+// If LED_PIN is not defined in "defines.h" take the default pin from "pins_arduino.h" framework.
+// If there is no pin defined for the board, use LED_PIN 255 and disable LED
+#if !defined(LED_PIN)
+    #if defined(LED_BUILTIN)
+        #define LED_PIN LED_BUILTIN
+        #define ENABLE_LEDS true
```

Comment:
What if `ENABLE_LEDS` is set to `false`?

Reply:
Good catch. I think i have to redo that whole part, it it should not get more complex.

Reply:
I changed some stuff.  Can you review again?
I added for every board. Ignoring ENABLE_LEDS  (overwriting it)
  #define LED_PIN LED_BUILTIN
  #define LED_INVERTED true